mamba/mamba/utils.py

```diff
@@ -211,6 +211,11 @@ def init_api_context(use_mamba_experimental=False):
     api_ctx.channels = context.channels
     api_ctx.platform = context.subdir
 
+    if "MAMBA_EXTRACT_THREADS" in os.environ:
+        max_threads = int(os.environ["MAMBA_EXTRACT_THREADS"])
```

Comment:
Why not put this straight into `context.hpp` as is done for a couple other default-to-env var options?

Reply:
we can but the `Context` value is not used except from the `Configuration` to set the semaphore max

Reply:
maybe we shoudl use it? When starting the downloads.